getting_started/step_by_step/your_first_game.rst

```diff
@@ -1231,16 +1219,17 @@ click "Groups" and you can type a new group name and click "Add".
 .. image:: img/group_tab.png
 
 Now all mobs will be in the "mobs" group. We can then add the following line to
-the ``game_over()`` function in ``Main``:
+the ``new_game()`` function in ``Main``:
```

Comment:
Why move this to `new_game()`?

Reply:
This way you can see the enemies after you lose.